README.md

```diff
@@ -137,8 +138,30 @@ Removing a version from the build won't make it be removed from Docker registry,
 
 ## Using and extending
 
+### PHP FPM healthcheck
+
+This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with the single process Docker container.
+
+This healthcheck provides diverse metrics to be watched upon and can be fitted accordingly to your needs, more information in how to use it in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).
+
+The healthcheck can be found in the container `$PATH` as executable:
+
+```console
+$ php-fpm-healthcheck
+$ echo $?
+0
+```
+
+## Basic usage
+
 Simply use the images as base of the application's `Dockerfile` and apply the necessary changes.
 
+```Dockerfile
+# syntax=docker/dockerfile:1.0.0-experimental
+
+FROM usabillabv/php:7.3-fpm-alpine3.9
+```
+
 In usual cases it might not be necessary to extend the nginx images, unless you desired extend it's behavior by for instance serving static files.
```

Comment:
```suggestion
In usual cases it might not be necessary to extend the nginx images, unless you desire to extend it's behavior by for instance serving static files.
```